business_requirement_deliverable_cost/models/business.py

```diff
@@ -257,5 +253,7 @@ def _compute_resource_procurement_total(self):
         'resource_procurement_total')
     def _compute_gross_profit(self):
         for br in self:
-            br.gross_profit = br.total_revenue - \
-                br.resource_task_total - br.resource_procurement_total
+            br.gross_profit = (
```

Comment:
Yes it's easier to read like this IMO (though you still need the parenthesis), unfortunately PEP8 doesn't force this syntax